src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -210,12 +212,12 @@ private AIState initiate()
             }
         }
 
-        BlockPosUtil.tryMoveLivingToXYZ(worker, job.getSchematic().getPosition());
+        BlockPosUtil.tryMoveLivingToXYZ(worker, job.getStructure().getPosition());
 
         return AIState.IDLE;
     }
 
-    private void loadSchematic()
+    private void loadStructure()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "loadStructure" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)